src/main/resources/templates/gcc/activities-next-class-schedule.html

```diff
@@ -15,7 +15,7 @@
             </div>
             <div class="form-card__footer">
               <th:block th:replace="~{fragments/continueButton :: continue}"/></br>
-              <a th:href="'/flow/' + ${flow} + '/activities-ed-program-dates'"
+              <a th:id="skip-coverage-during-classtime" th:href="'/flow/' + ${flow} + '/activities-ed-program-dates'"
```

Comment:
`th:id` is not needed
```suggestion
              <a id="skip-coverage-during-classtime" th:href="'/flow/' + ${flow} + '/activities-ed-program-dates'"
```